private-templates-service/client/src/components/Dashboard/RecentlyViewed/RecentlyViewed.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from "react";
+import { Template, PostedTemplate } from "adaptive-templating-service-typescript-node";
+import {
+  RecentlyViewedContainer,
+  RecentlyViewedBodyRow,
+  RecentlyViewedHeader,
+  RecentlyViewedItem,
+  RecentlyViewedBody,
+  RecentlyViewedHeaderItem
+} from "./styled";
+import { getDateString } from "../../../utils/versionUtils";
+import { StatusIndicator, Status, Title, TimeStamp } from "../PreviewModal/TemplateInfo/styled";
+import { TemplateStateWrapper } from "../../AdaptiveCardPanel/styled";
+
+interface Props {
+  recentlyViewed: Template[];
+  onClick?: (templateID: string) => void;
+}
+
+export class RecentlyViewed extends Component<Props> {
+  private constructRecentlyViewedRows(templates: Template[], propsOnClick: (id: string) => any): JSX.Element[] {
+    let rows: JSX.Element[] = new Array();
+    rows = templates.map(template => {
+      let onClick = () => {
+        if (propsOnClick && template.id) {
+          propsOnClick(template.id);
+        }
+      };
+      return (
+        <RecentlyViewedBodyRow onClick={onClick}>
+          <RecentlyViewedItem>{template.name}</RecentlyViewedItem>
+          <RecentlyViewedItem>{template.updatedAt ? getDateString(template.updatedAt) : "N/A"}</RecentlyViewedItem>
+          <RecentlyViewedItem>
+            <TemplateStateWrapper style={{ justifyContent: "flex-start" }}>
```

Comment:
![image](https://user-images.githubusercontent.com/21356912/76131972-6cfdf480-5fc5-11ea-9141-a983d824b094.png)
Because when TemplateState is displayed in the Recent Edited, to make it look like in the design, I used "center" to get it closer to the edge of the card (look at the screen). And at the same time in the Recently Viewed TemplateState needs to be in the beginning of its box to match column.
I've removed justify content from the TemplateStateWrapper and added individual ones to two different uses